src/qcodes/instrument_drivers/Keithley/Keithley_2450.py

```diff
@@ -132,7 +134,7 @@ def __exit__(
         self,
         exception_type: Optional[type[BaseException]],
         value: Optional[BaseException],
-        traceback: Optional[TracebackType],
+        traceback: Optional["TracebackType"],
```

Comment:
for the quotes usage - with move to python 3.10+ those will be converted to not-strings, right?

Reply:
@astafan8 No, once we move to 3.10+ we can use `|` for union types without ```from __future__ import annotations` but the forward references has not stabilized yet in any version and its not clear that it will and when that might be. We could add a `from __future__ import annotations` to all these files instead if you prefer?

Reply:
Does look like [pep649](https://peps.python.org/pep-0649/) has been accepted for 3.13 so this will probably eventually become redundant